tensorflow/contrib/signal/python/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Empty __init__.py files can either be completely empty, or they can contain license headers, py3 compatibility imports, and nothing else. I would prefer completely empty, but it doesn't matter much either way.